src/main/resources/db/changelog/changes/17-create-shopping_cart_cart_items.yaml

```diff
@@ -0,0 +1,37 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping_cart_cart_items
```

Comment:
this is redundant bacause we have a many-to-one relation and cart_item can just have a join column to store that relation